provider/openstack/networking_interfaces.go

```diff
@@ -68,6 +68,22 @@ type NetworkingNeutron interface {
 	ListFloatingIPsV2(...*neutron.Filter) ([]neutron.FloatingIPV2, error)
 	ListNetworksV2(...*neutron.Filter) ([]neutron.NetworkV2, error)
 	ListSubnetsV2() ([]neutron.SubnetV2, error)
+	ListSecurityGroupsV2(query neutron.ListSecurityGroupsV2Query) ([]neutron.SecurityGroupV2, error)
+	CreateSecurityGroupV2(name, description string, tags []string) (*neutron.SecurityGroupV2, error)
+	GetSubnetV2(subnetID string) (*neutron.SubnetV2, error)
+	GetFloatingIPV2(ipId string) (*neutron.FloatingIPV2, error)
+	DeleteFloatingIPV2(ipId string) error
+	PortByIdV2(portId string) (neutron.PortV2, error)
+	SecurityGroupByNameV2(name string) ([]neutron.SecurityGroupV2, error)
+	DeleteSecurityGroupV2(groupId string) error
```

Comment:
Hi Adis, couldnt find this in the PR context, why do we add some of these interface methods if we are not using them? Are we planning to implement them later?

Reply:
I see, thanks for clarifying :)